main/core.js

```diff
@@ -9,6 +9,7 @@ const fs = require('node:fs/promises')
 const JSONStream = require('jsonstream')
 const Sentry = require('@sentry/node')
 const consts = require('./consts')
+const { core } = require('@filecoin-station/core')
```

Comment:
As explained in a comment, I believe you need to load Core via dynamic import. Since CommonJS modules don't support top-level await, you will need to do that inside an async function.

```js
const {core} = await import('@filecoin-station/core');
```